ui/src/app/views/cohort-builder/search/wizard-modal/wizard-modal.component.ts

```diff
@@ -59,13 +59,15 @@ export class WizardModalComponent implements OnInit, OnDestroy {
   updateSearchResults() {
     this.updateOrCreateSearchResult();
 
-    this.searchGroupSubscription = this.searchService.getResults(
-      new SearchRequest(this.criteriaType.toUpperCase(), this.selectedSearchResult))
-      .subscribe(response => {
-        this.selectedSearchResult.updateWithResponse(response);
-        this.broadcastService.updateCounts(this.selectedSearchGroup, this.selectedSearchResult);
-        this.wizardModalRef.destroy();
-      });
+    //TODO: fix this when implementing this story
```

Comment:
I wonder if the implementor of that story will think to search for this. Might want to also include a comment in that JIRA ticket.

Reply:
Good idea.